src/promptflow/promptflow/_sdk/operations/_test_submitter.py

```diff
@@ -114,7 +116,10 @@ def _resolve_data(self, node_name: str = None, inputs: dict = None):
                     merged_inputs[name] = flow_inputs[name]
                 else:
                     if value.default is None:
-                        missing_inputs.append(name)
+                        if chat_history_name and name == chat_history_name:
```

Comment:
may add some comment on when will this happen. or it will be confusing

Reply:
Add comment to explain why set empty list to chat history